README.md

```diff
@@ -80,63 +76,63 @@ noteCount                      - int, note events count.
         SliderTail = 5, 
         BurstSliderHead = 6, 
         BurstSliderElement = 7
-  eventTime                    - float, song time of event 
-  spawnTime                    - float, spawn time of note
-  eventType                    - int, good = 0,bad = 1,miss = 2,bomb = 3
-  {                            - Cut info structure (only for Good and Bad!)
-    bool speedOK;
-    bool directionOK;
-    bool saberTypeOK;
-    bool wasCutTooSoon;
-    float saberSpeed;
-    Vector3 saberDir;
-    int saberType;
-    float timeDeviation;
-    float cutDirDeviation;
-    Vector3 cutPoint;
-    Vector3 cutNormal;
-    float cutDistanceToCenter;
-    float cutAngle;
-    float beforeCutRating;
-    float afterCutRating;
+  eventTime                    - float, song time of event.
+  spawnTime                    - float, spawn time of note.
+  eventType                    - int, good = 0, bad = 1, miss = 2, bomb = 3.
+  {                            - Cut info structure (only for Good and Bad!).
+    speedOK                    - bool, note was hit at passable speed.
+    directionOK                - bool, note was hit with correct direction.
+    saberTypeOK                - bool, note was hit with correct saber.
+    wasCutTooSoon              - bool, note was cut too soon.
+    saberSpeed                 - float, speed of the saber.
+    { x, y, z }                - 3 floats, direction of the saber.
+    saberType                  - int, type of the saber (0 = left, 1 = right).
+    timeDeviation              - float, how far away the time hit was from the time expected.
+    cutDirDeviation            - float, how far away the cut direction was from the one expected.
+    { x, y, z }                - 3 floats, point at which the note was cut.
+    { x, y, z }                - 3 floats, cut normal vector.
+    cutDistanceToCenter        - float, distance from center of the cut.
+    cutAngle                   - float, angle of the cut.
+    beforeCutRating            - float.
```

Comment:
Angle rating before the cut, with after cut and cut distance forming 115 score. This beforeCutRating is uncapped (can go over 1) 1 means 70 score.